src/ankit/MathFunctionsWhile.java

```diff
@@ -0,0 +1,129 @@
+package ankit;
+
+public class MathFunctionsWhile {
+int startIndex, endIndex;
+
+	void evenNumber(int startIndex,int endIndex) {
+		if(startIndex == endIndex) {
+			System.out.println("Both numbers are same please provide a vaild range");
+		}
+		else if (startIndex < endIndex) {
+			 int index=startIndex;
```

Comment:
you don’t require to create extra variable, you can handle this without **index**, think on this

please refer below code
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/208/files
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/219/files